package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "what-to-watch",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@emotion/react": "^11.9.0",
+    "@emotion/styled": "^11.8.1",
+    "@mui/material": "^5.8.0",
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^13.0.0",
+    "@testing-library/user-event": "^13.2.1",
+    "@types/jest": "^27.0.1",
+    "@types/node": "^16.7.13",
+    "@types/react": "^18.0.0",
+    "@types/react-dom": "^18.0.0",
+    "axios": "^0.27.2",
+    "emmet": "^2.3.6",
```

Comment:
Why do you added emmet?

Reply:
I thought we discussed this at the meeting. Perhaps I misunderstood. In general, a useful thing in my opinion.